axelrod/strategies/shortmem.py

```diff
@@ -0,0 +1,50 @@
+from axelrod import Actions, Player
+from axelrod.actions import Action
+
+C, D = Actions.C, Actions.D
+
+class ShortMem(Player):
+    """
+    A player starts by always cooperating for the first 10 moves.
+    
+    The opponent answers are stored in the memory, whose maximum size is 10
```

Comment:
Technically all the history is saved in memory in our library. I would just say that the player analyzes the last ten rounds of play without any details about what's in memory.

answers -> actions